apps/web/src/components/SelfDeclaration/CommunitySelection.tsx

```diff
@@ -93,129 +80,150 @@ export const CommunityList = ({ labels }: CommunityListProps) => {
     }
   };
 
-  return (
+  const firstNations = (
```

Comment:
Can we move these out into components? While this should be fine, it is dangerously close to [a common pitfall](https://react.dev/learn/your-first-component#:~:text=you%20must%20never%20nest%20their%20definitions) in react.

Reply:
a85d30f4bfad44c3007ac9fb762091bc606889fc